designs/limits.md

```diff
@@ -0,0 +1,60 @@
+# Karpenter Limits
+
+This document proposes an approach to limit the scaling that Karpenter will perform, in order to control customer costs and to prevent underutilized capacity in cases where pending pods cannot be scheduled on new capacity.
```

Comment:
How does this doc address "underutilized capacity" ?

Reply:
It does not, let me fix this. I originally meant to discuss more about the runaway-scaling problem where your instances are effectively underutilized since none of your pods are running but ended up not doing that